static/css/base.scss

```diff
@@ -164,3 +164,49 @@ tbody a {
 .card-title {
   font-size: 1.5em;
 }
+
+// year-calendar
+.calendar .calendar-header {
+  background-color: #ed174b;
```

Comment:
Variabeln wie Farben würde ich unbedingt in ein variables scss schreiben und importieren. Ansonsten wird es mega schwierig irgendwo eine Änderung vorzunehmen!